bundles/org.openhab.binding.yioremote/README.md

```diff
@@ -0,0 +1,59 @@
+# YIOremote Binding
+
+This binding will control a YIO Dock/Remote combination. YIO Remote/Dock is a smart home solution that includes an IP based remote. More information can be found at [yio-remote](https://www.yio-remote.com/) or in the forums at [yio-remote](https://community.yio-remote.com/). 
+
+This binding has been designed to compliment the YIO websocket Transport Protocol.
+
+Since this binding allows actual you to trigger IR send/receive actions on YIO Dock, this allows you to use the YIO Dock as an IR solution to openHAB and even learn new IR codes from your remotes. In other words, if the IR code is known then openHAB can use the YIO Dock to control that Device regardless if there is an openHAB binding for it or not.
+
+## Supported Things
+
+* Thing: YIO Dock.
+
+The following are the configurations available to each of the bridges/things:
+
+### YIO Dock
+
+| Name                 | Type    | Required | Default | Description                                                                                                    |
+|----------------------|---------|----------|---------|----------------------------------------------------------------------------------------------------------------|
+| yiodockhost	       | string  | Yes      | (None)  | Host/IP Address or host name of the YIO Dock                                                                   |
```

Comment:
The "yiodock" prefix could be removed as it is redundant for all config parameters/channels.